assets/js/gathercontent.js

```diff
@@ -79,6 +79,18 @@ window.GatherContent = window.GatherContent || {};
 		}
 	};
 
+	main.maybeRevealComponent = function (evt) {
```

Comment:
For our future selves why is this `maybe` reveal component? Might be good to explain what this is doing. I appreciate it's a copy of the original `maybeReveal` but adding the context now will be really valuable.